README.md

```diff
@@ -1,9 +1 @@
-# About this Repo
-
-[![Build Status](https://travis-ci.org/appropriate/docker-jetty.svg?branch=master)](https://travis-ci.org/appropriate/docker-jetty)
-
-This is the Git repo of the official Docker image for [jetty](https://registry.hub.docker.com/_/jetty/). See the
-Hub page for the full readme on how to use the Docker image and for information regarding contributing and issues.
-
-The full readme is generated over in [docker-library/docs](https://github.com/docker-library/docs),
-specificially in [docker-library/docs/jetty](https://github.com/docker-library/docs/tree/master/jetty).
+# Repository moved to https://github.com/eclipse/jetty.docker 
```

Comment:
I think this should be a bit better than a half sentence.

By all means lead with a big headline about how the repo has moved.    But then have a paragraph about what the repo is, how it was started by appropriate and then moved to co-reside with the jetty project at eclipse.